combat-trainer.lic

```diff
@@ -20,6 +20,12 @@ class SetupProcess
     @cycle_armors = settings.cycle_armors
     echo("  @cycle_armors: #{@cycle_armors}") if $debug_mode_ct
     @last_cycle_time = Time.now - 125
+    
+    @warhorn = settings.warhorn
+    echo("  @warhorn: #{@warhorn}") if $debug_mode_ct
+    @use_warhorn = settings.use_warhorn
```

Comment:
No need for use_warhorn setting. This would just need to be 

`@warhorn` and then the check inside the methods would be `if @warhorn`

With how ruby doesn't know booleans, just checking if it exists or not is enough. If the warhorn is named, then it knows to use a warhorn.

Reply:
done